### ISSUE TITLE
a bug in dis.py

### ISSUE THREAD

**Issue by wdyxwzyh** [princewen/tensorflow_practice#42]
emm  in dis.py.131, the L2 regr has been add to calculate D loss , and maybe the equation is wrong?
        self.loss = tf.reduce_mean(losses) + self.l2_reg_lambda + self.l2_loss
    it should be:
        self.loss = tf.reduce_mean(losses) + self.l2_reg_lambda * self.l2_loss

**Comment by wdyxwzyh** [comment]
also in train.py.110,  there should be generator.pgen_loss_adv instead of generator.gen_loss_adv
